internal/webhook/external/github.go

```diff
@@ -92,7 +96,7 @@ func (g *githubWebhookReceiver) getHandler(requestBody []byte) http.HandlerFunc
 
 		eventType := r.Header.Get(gh.EventTypeHeader)
 		switch eventType {
-		case githubEventTypePackage, githubEventTypePing, githubEventTypePush:
+		case githubEventTypePackage, githubEventTypePing, githubEventTypePush, githubEventTypeRegistryPackage:
```

Comment:
Just slightly more readable than the long line:

```suggestion
		case githubEventTypePackage,
			githubEventTypePing,
			githubEventTypePush,
			githubEventTypeRegistryPackage:
```